Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Screens/IOS/Alert.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium.Appium;
+using System;
+
+namespace Aquality.Appium.Mobile.Template.Screens.IOS
+{
+    public class Alert : IOSScreen, Interfaces.IAlert
```

Comment:
can we add using of `Interfaces` namespace?

Reply:
There is a conflict with Selenium.IAlert interface

Reply:
using IAlert = Aquality.Appium.Mobile.Template.Screens.Interfaces.IAlert?
Or you can just rename your interface to smth less common